recipes/loguru/all/conanfile.py

```diff
@@ -0,0 +1,147 @@
+        "filename_width": "ANY",        # int
+        "threadname_width": "ANY",      # int
+        "scope_time_precision": "ANY",  # int
+    }
+
+    default_options = {
+        "use_fmtlib": False,
+        "shared": False,
+        "catch_sigabrt": True,
+        "verbose_scope_endings": True,
+        "redefine_assert": False,
+        "with_streams": False,
+        "with_fileabs": False,
+        "with_rtti": True,
+        "replace_glog": False,
+
+        "scope_text_size": None,         # default: 196
+        "filename_width": None,          # default: 23
+        "threadname_width": None,        # default: 16
+        "scope_time_precision": None,    # 3=ms, 6=us, 9=ns, default: 3
+    }
+
+    def build_requirements(self):
+        self.tool_requires("cmake/3.25.0")
+
+    def requirements(self):
+        if self.options.use_fmtlib:
+            self.requires("fmt/9.1.0", transitive_headers=True)
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, 11)
+
+        if self.options.replace_glog:
+            print("loguru: replace_glog in effect, forcing with_streams=True")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+
+    def layout(self):
+        cmake_layout(self)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+
+        tc.variables["LOGURU_USE_FMTLIB"] = self.options.use_fmtlib
+        tc.variables["LOGURU_STATIC"] = not self.options.shared
+        tc.variables["LOGURU_CATCH_SIGABRT"] = _int(self.options.catch_sigabrt)
+        tc.variables["LOGURU_VERBOSE_SCOPE_ENDINGS"] = _int(self.options.verbose_scope_endings)
+        tc.variables["LOGURU_REDEFINE_ASSERT"] = _int(self.options.redefine_assert)
+        tc.variables["LOGURU_WITH_STREAMS"] = _int(self.options.with_streams)
+        tc.variables["LOGURU_WITH_FILEABS"] = _int(self.options.with_fileabs)
+        tc.variables["LOGURU_WITH_RTTI"] = _int(self.options.with_rtti)
+        tc.variables["LOGURU_REPLACE_GLOG"] = _int(self.options.replace_glog)
+
+        if self.options.scope_text_size:
+            tc.variables["LOGURU_SCOPE_TEXT_SIZE"] = self.options.scope_text_size
+        if self.options.filename_width:
+            tc.variables["LOGURU_FILENAME_WIDTH"] = self.options.filename_width
+        if self.options.threadname_width:
+            tc.variables["LOGURU_THREADNAME_WIDTH"] = self.options.threadname_width
+        if self.options.scope_time_precision:
+            tc.variables["LOGURU_SCOPE_TIME_PRECISION"] = self.options.scope_time_precision
+
+        tc.generate()
+        deps = CMakeDeps(self)
+        deps.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, pattern='LICENSE.md', src=self.source_folder, dst=os.path.join(self.package_folder, 'licenses'))
```

Comment:
```suggestion
        copy(self, pattern='LICENSE.md', src=self.source_folder, dst=os.path.join(self.package_folder, 'licenses'))
```
extract it from the source file instead. https://docs.conan.io/en/latest/howtos/extract_licenses_from_headers.html